src/screens/LegalRights/VideoPage/index.tsx

```diff
@@ -1,12 +1,69 @@
+import { useEffect, useRef, useState } from 'react';
 import { Pressable, ScrollView, Text, View } from 'react-native';
 import { Video } from 'expo-av';
+import supabase from '@/supabase/createClient';
 import { styles } from './styles';
 
-export default function VideoPage() {
+export default function VideoPage(testProp: any) {
+  // var for array of all the pages for the current language
+  const [preaData, setPreaData] = useState([
+    {
+      id: 'string',
+      is_short_answer: true,
+      page_number: 0,
+      parent_id: 'string',
+      short_answer: 'string',
+      spanish: false,
+      survey: 'string',
+      video_id: 'Section Title 1',
+    },
+  ]);
```

Comment:
try to think about what data you *actually* need in this page, also look into `Typescript` interfaces so we don't use `any` too often